### ISSUE TITLE
NumericLiterals auto-gen-config Off By One

### ISSUE THREAD

**Issue by tmorris-fiksu** [rubocop/rubocop#884]
`rubocop --auto-gen-config` does not correctly repair the NumericLiterals cop.
For a 6 digit number, the following is generated:

```
NumericLiterals:
  MinDigits: 6
```

Either the cop should ignore 6 digit numbers or the cop's auto-gen should be 7.

Sample file and version:

``` bash
$ cat sample.rb
y = 123456
puts y
$ rubocop -V
0.19.0 (using Parser 2.1.7, running on ruby 1.9.3 x86_64-darwin12.5.0)
```
